src/MessageBox.Avalonia/Extensions/OpenUrl.cs

```diff
@@ -19,10 +19,8 @@ public static void OpenUrl(this string url)
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                 {
                     //https://stackoverflow.com/a/2796367/241446
-                    using (Process proc = new Process { StartInfo = { UseShellExecute = true, FileName = url } })
-                    {
-                        proc.Start();
-                    }
+                    using Process proc = new() { StartInfo = { UseShellExecute = true, FileName = url } };
```

Comment:
Looks like I didnt see this, but anyway I will do changes later

Reply:
ok have fixed